### PR TITLE
Initial Statistics Page, more features to come

### PR DESCRIPTION
**Pull request by SunnyNagam**
image

### CODE REVIEW

**Comment by brian-norman** [comment]
Bugs:
1. Sunny's score dissapears when normalizing
2. My scores don't seem to change
3. Lets make sure we don't ship the giant json

Nits:
1. "normalise" -> "normalize"

File: netlify/functions/movie.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```
Comment (cole-adams) [inline netlify/functions/movie.ts:1]: This eslint disable doesn't look like its needed

File: netlify/functions/movie.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    context: HandlerContext
+    ): Promise<HandlerResponse> {
+        if (event.httpMethod === 'GET') {
+            const pathArray = event.path.split("/");
+            const movieIndex = pathArray.indexOf('movie') + 1;
+            const movieId = parseInt(pathArray[movieIndex]);
+            
+            if (isNaN(movieId)) {
+                return {
+                    statusCode: 422,
```
Comment (cole-adams) [inline netlify/functions/movie.ts:36]: The club endpoint uses a 404 not found response for invalid clubId, I think we should do the same here

File: src/views/StatisticsView.vue [hunk 3]
```diff
@@ -1,3 +1,550 @@
...
+      }
+    }
+  }
+  headers.push({value: "averageNorm"});
+  headers.push({value: "vote_average", title:"TMDB"});
+  return headers;
+});
+</script>
+
+<style scoped>
```
Comment (cole-adams) [inline src/views/StatisticsView.vue:519]: All our styles are now handled by Tailwind. We should probably do the same here

File: src/views/StatisticsView.vue [hunk 4]
```diff
@@ -1,3 +1,550 @@
...
+      fillOpacity: 0,
+      strokeWidth: 3,
+      stroke: 'red',
+      bins: bins,
+      showInLegend: false
+    },
+    {
+      type: 'histogram',
+      xKey: 'cole',
+      xName: 'Cole Score',
```
Comment (cole-adams) [inline src/views/StatisticsView.vue:380]: Does this section need to be abstracted for other clubs?

**Comment by brian-norman** [comment]
Can’t really differentiate x axis and y axis right now, can we add labels?

**Review by cole-adams — APPROVED** [review]

**Pull request merged**